blazor/predefined-dialogs/getting-started.md

```diff
@@ -0,0 +1,334 @@
+---
+layout: post
+title: Getting Started with Blazor Predefined Dialogs Component | Syncfusion
+description: Checkout and learn about getting started with Blazor Predefined Dialogs component in Blazor Server App and Blazor WebAssembly App.
```

Comment:
Description: Check out and learn about getting started with the Blazor Predefined Dialogs component in the Blazor Server App and Blazor WebAssembly App.